src/Toml/Bi/Combinators.hs

```diff
@@ -513,3 +513,33 @@ map keyCodec valCodec key = Codec input output
                     insertTableArrays key $ t :| (ts ++ tomls)
 
         dict <$ modify updateAction
+
+tableMapCodec
+    :: Ord key
+    => TomlBiMap Key key  -- ^ Bidirectional converter between TOML and Map keys
+    -> TomlBiMap val AnyValue  -- ^ Bidirectional converter between TOML and Map values
```

Comment:
Yeah, that's true that this function need to take a `TomlBiMap` instead of `Codec` since it works on a level of a single _key-value_ pair, instead of multiple keys. Good catch!